.github/workflows/docker-image.yml

```diff
@@ -45,12 +55,77 @@ jobs:
     - name: Build & push docker images
       env:
         ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-        IMAGE_TAG: ${{ steps.date.outputs.date }}
+        IMAGE_TAG: ${{ steps.get_version.outputs.VERSION }}
       run: |
         docker build -f Dockerfile -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
         docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
     
     - name: Logout of Amazon ECR
       if: always()
       run: docker logout ${{ steps.login-ecr.outputs.registry }}
+
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Deploy to elia_staging
+      env:
+          IMAGE_TAG: ${{ steps.get_version.outputs.VERSION }}
+
+      uses: peterkimzz/aws-ssm-send-command@master
+      id: ssm
+      with:
+        aws-region: eu-central-1
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        instance-ids: ${{ secrets.INSTANCE_ID }}
+
+        working-directory: /home/ubuntu/docker-stack/${{ github.event.repository.name }}
+        command: ls -la
+        command: echo $USER
+        
+        #Update code
+        command: git pull
+        
+        #Setup aws Credentials/Variables
+
+        #Remove old values
+        command: rm .env
+
+        #Export AWS credentials
+        command: export AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }}
+        command: export AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+        #Set up rest of variables
+        command: touch .env
+        command: echo AWS_DEFAULT_REGION=eu-central-1 >> .env
+        command: echo IMAGE_TAG=$IMAGE_TAG >> .env
+        command: echo AWS_ACCOUNT_ID_STAGING=${{ secrets.AWS_ACCOUNT_ID_STAGING }} >> .env
+        command: echo CPO_PRIVATE_KEY=${{ secrets.CPO_PRIVATE_KEY }} >> .env
+        command: echo CPO_TOKEN_A=${{ secrets.CPO_TOKEN_A }} >> .env
+        command: echo AWS_REPOSITORY_URL=$AWS_REPOSITORY_URL >> .env
+
+        #remove nginx certificates and key
+        command: rm ./${{ github.event.repository.name }}/devops/nginx/origin.crt
```

Comment:
Is this step necessary?
You override these files in next step

Reply:
true, will change

Reply:
Fixed pipe failure case